### PR TITLE
New Service class, incl. Jupyter integration

### PR DESCRIPTION
**Pull request by m-mohr**
Based on #206, which is split into three PRs.

- Adds new Service class
- Adds rich HTML rendering for Services
- ~ToDo: This has also code for service logs, but this needs #214 to be merged before!~

### CODE REVIEW

**Comment by soxofaan** [comment]
#214  was merged, so this PR can continue

**Comment by m-mohr** [comment]
This should be good to review / go. I've included the Service Logs again, but I have trouble with Jupyter right now so unable to test.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#204](https://github.com/Open-EO/openeo-python-client/issues/204))
 - Add `optional` argument to `Parameter` and fix re-encoding parameters with default value. (EP-3846)
 - Add support to test strict equality with `ComparableVersion`
-- Jupyter integration: add rich HTML rendering for more backend metadata (Job, Job Estimate, Logs, User-Defined Processes)
+- Jupyter integration: add rich HTML rendering for more backend metadata ()
```
Comment (soxofaan) [inline CHANGELOG.md:17]: This line looks out of place

Comment (m-mohr) [reply CHANGELOG.md:17]:
```suggestion
```

Comment (m-mohr) [reply CHANGELOG.md]: Oh yes, come copy&paste magic happened there.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,7 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#204](https://github.com/Open-EO/openeo-python-client/issues/204))
 - Add `optional` argument to `Parameter` and fix re-encoding parameters with default value. (EP-3846)
 - Add support to test strict equality with `ComparableVersion`
-- Jupyter integration: add rich HTML rendering for more backend metadata (Job, Job Estimate, Logs, User-Defined Processes)
+- Jupyter integration: add rich HTML rendering for more backend metadata (Job, Job Estimate, Logs, Services, User-Defined Processes)
+- Added class `Service` for secondary web-services instead of returning a `dict` for them
```
Comment (soxofaan) [inline CHANGELOG.md:18]: This should go under "Changed" I think, and be more explicit that the return type of `tiled_viewing_service`/`create_service` changed

Comment (m-mohr) [reply CHANGELOG.md]: Okay, I've added more entries to the changelog, it should really be complete now.

**Comment by soxofaan** [comment]
rebased, squashed and merged in f42ba7f

**Comment by soxofaan** [comment]
thanks!

**Pull request closed without merging**